### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by Beanow**
Rather than a proposal, this adopts the proposal for SFOSC and starts using it.

### CODE REVIEW

**Comment by Beanow** [comment]
Can you share your thoughts on these workgroups?
(The generation code and pages are prototype, we'll flesh that out over more steps)

**Comment by Beanow** [comment]
I see there's still a bunch of SFOSC work in the readme. I'll look into that.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,28 @@
 # SFOSC workgroups
```
Comment (nothingismagick) [inline README.md]: here

**Comment by Beanow** [comment]
Setting up social media accounts under wg-organization seemed really weird. So I'm adding wg-media here.

**Pull request merged**